File: 242-ValidAnagram/242-ValidAnagram.java
// Last updated: 8/7/2025, 2:56:59 PM
class Solution {
    public boolean isAnagram(String s, String t)
    {
        int f1[]=new int[26];
        int f2[]=new int[26];
        for(int i=0;i<s.length();i++)
            f1[s.charAt(i)-'a']++;
        for(int i=0;i<t.length();i++)
            f2[t.charAt(i)-'a']++;
        for(int i=0;i<26;i++)
        {
            if(f1[i]!=f2[i]) return false;
        }
        return true;
    }
}

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 8/7/2025, 2:57:04 PM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        f=[0]*26
        k=[0]*26
        for i in s:
            #print(ord('a'))
            f[ord(i)-97]+=1
        #print(f)
        for i in t:
            #print(ord('a'))
            k[ord(i)-97]+=1
        #print(f)
        if k==f:
            return True
        else:
            return False
        